src/Compilers/CSharp/Portable/Binder/AwaitableInfo.cs

```diff
@@ -49,15 +49,5 @@ public override int GetHashCode()
         {
             return Hash.Combine(GetAwaiter, Hash.Combine(IsCompleted, Hash.Combine(GetResult, Type.GetHashCode())));
         }
-
-        public static bool operator ==(AwaitableInfo left, AwaitableInfo right)
-        {
-            return left.Equals(right);
-        }
-
-        public static bool operator !=(AwaitableInfo left, AwaitableInfo right)
-        {
-            return !left.Equals(right);
-        }
```

Comment:
If `operator==` is removed, the `BoundNode.Update` methods will use `ReferenceEquals`. If that's reasonable, then can the `Equals` methods be removed from this class as well?

Reply:
I think reference equality would be sufficient. We can drop `Equals`, `IEquatable` and `GetHashCode`.